### ISSUE TITLE
If an error occurs when embedding an image, display the alt text instead

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-pdf#693]
If an error occurs while attempting to embed an image into the PDF, display the alt text instead (as well as the caption, if one is present). Currently, only the caption is shown. This makes the behavior inconsistent with that of preemptive failures (such as when the image file is missing).

Verify this behavior for both block and inline images.

**Comment by mojavelinux** [comment]
It might make sense to introduce a `layout_alt_text` helper method. The alternative is to move the begin/rescue block to the method and use an exception to control the flow.
